packages/block-library/src/embed/core-embeds.js

```diff
@@ -292,8 +293,9 @@ export const others = [
 			title: 'SmugMug',
 			icon: embedPhotoIcon,
 			description: __( 'Embed SmugMug content.' ),
+			previewable: false,
 		},
-		patterns: [ /^https?:\/\/(www\.)?smugmug\.com\/.+/i ],
+		patterns: [ /^https?:\/\/(.+\.)?smugmug\.com\/.*/i ],
```

Comment:
Is this a desired change?

Reply:
I believe so, remembering back to #11960 it uses personalized subdomains like `https://johndavis.smugmug.com/Nature/Landscape/i-NntLPZ8`

Reply:
This is related to (from the original comment):

> An interesting consequence of the fact that these were maintained separately is that the pattern for SmugMug URLs was inaccurate since it didn't capture subdomains (and likewise did not match the [equivalent pattern defined in PHP](https://github.com/WordPress/wordpress-develop/blob/98bee9bf081e2c43c8908002d75870c3dfd61b19/src/wp-includes/class-oembed.php#L61)).

The new pattern is meant to precisely match the one used in core:

https://github.com/WordPress/wordpress-develop/blob/b0889a7/src/wp-includes/class-wp-oembed.php#L61

Reply:
Noting that this was updated separately in #21744.